scratch/collect_items.py

```
@@ -0,0 +1,40 @@
+    "スキンケア",
+    "文房具",
+    "ファッション",
+    "Amazonデバイス",
+    "ゲーム"
+]
+
+results = []
+
+for cat in categories:
+    print(f"Searching for {cat}...")
+    subprocess.run(["uv", "run", "python", "scripts/creators_search_items.py", cat])
```

Ruff (0.15.17)

[error] `subprocess` call: check for execution of untrusted input

(S603)